source/docs/v3/integrations/spatie.blade.md

```diff
@@ -8,6 +8,9 @@ section: content
 
 ## **laravel-activitylog** {#laravel-activitylog}
 
+> Note: The ID of the model who's the subject of Activity needs to be an integer. We recommend extra security measures when using integer IDs for tenants. Because the IDs become enumerable, they get vulnerable to enumeration attacks (which UUIDs are safe against).
```

Comment:
> who

models are inanimate

And the first sentence seems pretty clunky and complex

Reply:
```suggestion
> Note: The model needs to have an integer ID to have activity. We recommend extra security measures when using integer IDs for tenants. Because the IDs become enumerable, they get vulnerable to enumeration attacks (which UUIDs are safe against).
```

Is this better?

Reply:
> to have activity

is weird

Reply:
I mean, you don't have to write about performing activity or having activity. You should just explain the package's requirements. This isn't about needing an integer ID to perform activity. It's about the activity package requiring integer IDs. So just explain that.

Reply:
I'd go with the latter suggestion. If you want to make it more accurate, you could make it say something like "The package requires **logged** models to have integer IDs" (without the emphasis).

Reply:
"logged models" seems the best